### DISCUSSION TITLE
Save attachments in Nextcloud

### DISCUSSION THREAD

**Discussion by D1irk** [the-djmaze/snappymail#773] [Q&A]
Hi. I read somewhere that it is possible to save e-mail attachments in a Nextcloud folder, but I can't figure out how to do this. I just can open the file and/or download it to my local drive.
Nextcloud 25.0.2 with SnappyMail 2.23.0
Thanks, Dirk

**Comment by cm-schl** [comment]
Hi,

there is an open (but not answered by the opener) issue: https://github.com/the-djmaze/snappymail/issues/761

You should also check if the Nextcloud Plugin is active in the Admin Panel.

**Comment by D1irk** [comment]
Hi,

I doublechecked my settings as described in issue #761, but neither the "attachment nextcloud" button nor the save to Nextcloud functionally is available. 

What do you mean with 
> check if the Nextcloud Plugin is active in the Admin Panel?

I installed SnappyMail as an App in Nextcloud. The SnappyMail admin panel lists Extensions, but no plugins.

**Reply by MI-KY** [reply]
Yes, you're right - have a look into the extensions. There a "Nextcloud" extension should be installed and activated. This extension should handle the additional functionalities you're looking for...

**Reply by D1irk** [reply]
I don't see a Nextcloud extension in my installation. Activated is "Change Password", and additionally available are Avatars, Change Password Poppassd, Contacts suggestions (LDAP) and LDAP Mail Accounts. There is also no option to upload or search for other extensions.

**Reply by cm-schl** [reply]
🤔 you're right - there seams to be some problem with the plugin repository...
Created issue #778

**Reply by cm-schl** [reply]
Hi,

@the-djmaze solved the problem super fast and in the next release the problem should be solved (normally new releases are published also fast, so don't worry 🙂).
Because your install happened "in the wrong moment" after the update I would try to activate manually the extension "Nextcloud". If you don't see it anyway or have other problems I would try to remove the app in Nextcloud and reinstall it 🙂 .

**Reply by D1irk** [reply]
Hey guys, you are great :smile::thumbsup:  I just updated the App to v2.23.1 in Nextcloud and now it works as expected.
